packages/uilib/src/lib/plugins/communication-explorer/sidebar/sidebar.svelte

```diff
@@ -77,18 +95,31 @@
                     I don't think it adds much user value
                     and it is going to be hard to support
                 -->
+                <input
+                    type="text"
+                    placeholder="Filter IED"
+                    bind:value={searchQuery}
+                    on:input={handleIEDSearch}
+                />
                 <select
```

Comment:
Maybe we could refactor this as well with a svelte MUI component ?

Reply:
Please see: https://github.com/sprinteins/oscd-plugins/pull/46#discussion_r1793123695